src/file.js

```diff
@@ -12,35 +12,76 @@ var utils = require('./utils');
 
 var file = module.exports;
 
+/**
+ * Asynchronously copies `src` to `dest`
+ * @param {string} src - source filename to copy
+ * @param {string} dest - destination filename of the copy operation
+ * @param {function(Error): void} callback
+ */
+function copyFile(src, dest, callback) {
+	var calledBack = false;
+
+	var read;
+	var write;
+
+	function done(err) {
+		if (calledBack) {
+			return;
+		}
+		calledBack = true;
+
+		if (err) {
+			if (read) {
+				read.destroy();
+			}
+			if (write) {
+				write.destroy();
+			}
+		}
+
+		callback(err);
+	}
+
+	read = fs.createReadStream(src);
+	read.on('error', done);
+
+	write = fs.createWriteStream(dest);
+	write.on('error', done);
+	write.on('close', function () {
+		done();
+	});
+
+	read.pipe(write);
+}
+
+file.copyFile = (typeof fs.copyFile === 'function') ? fs.copyFile : copyFile;
+
 file.saveFileToLocal = function (filename, folder, tempPath, callback) {
 	/*
 	 * remarkable doesn't allow spaces in hyperlinks, once that's fixed, remove this.
 	 */
-	filename = filename.split('.');
-	filename.forEach(function (name, idx) {
-		filename[idx] = utils.slugify(name);
-	});
-	filename = filename.join('.');
```

Comment:
Please test to see if this block is still required. We don't even use remarkable anymore...

Reply:
... nope it does not. I'll commit a fix separately.

Reply:
It's still there, I just changed it to use `.map` instead